### DISCUSSION TITLE
Any chance to access the CSV dialect and delimiter options when exporting?

### DISCUSSION THREAD

**Discussion by sbv-csis** [simonw/datasette#1095] [Q&A]
I just got a request to export a tab separated file from my data and I was wondering if there was a way in general to tweak the csv formatting and dialect options?

https://docs.python.org/3/library/csv.html#csv-fmt-params

**Comment by simonw** [comment]
You can use the https://github.com/simonw/datasette-copyable plugin to get TSV export, but I agree with you: TSV is a common enough format that it would make sense for Datasette to support it out of the box.

**Reply by simonw** [reply]
Filed an issue here: https://github.com/simonw/datasette/issues/1096

**Comment by simonw** [comment]
Are there any other dialect options people would find useful? Any argument for adding something like a `?_delimiter=;` querystring argument?
